### PR TITLE
chore: add calendly to book a call and dob input pattern changes

### PR DESCRIPTION
**Pull request by iagormoraes**
[Ticket]()

## Summary
This PR includes adding the link for the "Book a Call" that is placed at the bottom of the layout and formatting the date input to MMDDYYYY.

## Changes
- updated layout
- updated DoB input pattern

## Testing
Tested locally.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [ ] I have made any relevant corresponding changes to the documentation including the project readme.
- [x] I have run and tested the changes locally
- [ ] If it is a core feature, I have added an appropriate amount of unit tests.
- [x] Any dependent changes have been merged and published in upstream projects https://github.com/VerifiedInc/Helm/pull/572

### CODE REVIEW

File: app/utils/date.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const dateUtils = {
+  /**
+   * Convert date from YYYY-MM-DD to DD-MM-YYYY
```
Comment (rsmets) [inline app/utils/date.ts:3]:
```suggestion
   * Convert date from MM-DD-YYYY to YYYY-MM-DD
```

File: app/components/BookACallButton.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import { VerifiedIncLogo } from './VerifiedIncLogo';
+
+type BookACallButtonProps = ButtonProps;
+
+export function BookACallButton(props: BookACallButtonProps) {
+  const theme = useTheme() as unknown as any;
+  const defaultProps = theme.components.MuiButton.defaultProps;
+  const href =
+    'https://calendly.com/d/5fg-xmg-9r6/verified-inc-intro-call?text_color=000000&primary_color=0dbc3d';
```
Comment (rsmets) [inline app/components/BookACallButton.tsx:12]: Would it be better if this url was an env var instead of hard coded?

**Review by rsmets — CHANGES_REQUESTED** [review]

File: app/utils/date.ts (hunk 1 above)
Comment (iagormoraes) [reply app/utils/date.ts:3]: Thanks for spotting this error.

File: app/components/BookACallButton.tsx (hunk 2 above)
Comment (iagormoraes) [reply app/components/BookACallButton.tsx:12]: I completely agree, and since we have application context, we can easily consume from it.

**Review by rsmets — APPROVED** [review]

**Pull request merged**